gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -75,7 +72,7 @@ def get_sample_age(sv_list: hl.Table) -> hl.Table:
     """
     sample_meta = meta.ht().key_by()
 
-    # NOTE: some 1KG samples were already in v3.0 (SV data) and were given a new prefix in v3.1(meta)
+    # NOTE: some 1KG samples were already in v3.0 (SV data) and were given a new prefix in v3.1 (meta)
```

Comment:
basic question, does this note need the `# noqa` annotation?

Reply:
Yeah, I can make this 88